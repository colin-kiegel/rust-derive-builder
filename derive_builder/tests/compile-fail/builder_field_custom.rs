#[macro_use]
extern crate derive_builder;

#[derive(Debug, PartialEq, Default, Builder, Clone)]
pub struct Lorem {
    #[builder(default = "88", field(type = "usize", build = "self.ipsum + 42"))]
    ipsum: usize,
}

fn main() {}
