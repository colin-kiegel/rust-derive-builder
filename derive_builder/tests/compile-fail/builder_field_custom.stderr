error: #[builder(default)] and #[builder(field(build="..."))] cannot be used together
 --> tests/compile-fail/builder_field_custom.rs:6:25
  |
6 |     #[builder(default = "88", field(type = "usize", build = "self.ipsum + 42"))]
  |                         ^^^^
